.github/workflows/release-baseci-testcontainers.yml

```
@@ -0,0 +1,36 @@
+name: Release BaseCI Testcontainers
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "baseci/*"
```

you need to update this 
```suggestion
      - "baseci-testcontainers/*"
```

---

Ignore this you've already done it